re-parent and update delegates if they are lost during keyboard initi…

jacobbullock opened the pull request:
I'd like to do some more digging to see why this is actually happening, but I've tried quite a few things that have turned up nothing.  This fixes the keyboard so that it should never appear blank when being used system wide.

gabrielwong commented on ios/engine/KMEI/KeymanEngine/Classes/InputViewController.swift:
```diff
@@ -134,6 +134,13 @@ open class InputViewController: UIInputViewController, KeymanWebViewDelegate {
     inputView?.setNeedsUpdateConstraints()
+    
+    //TODO: find out why this is actually happening
+    if(self.containerView?.subviews.count == 0)
```
`containerView` can be referenced without `self` and `isEmpty` can be used in place of `count == 0`.

jacobbullock commented:
@gabrielwong I fixed the lint warnings and updated the other preferred code styles you mention.  I am using an `if let` to unwrap containerView or checking for `.count == 0` requires a force unwrap and that gets ugly.
